### ISSUE TITLE
Implement a learning rate finder

### ISSUE THREAD

**Issue by flinder** [SMAPPNYU/stance_classification_experiment#21]

**Comment by vishakhpk** [comment]
To clear up the question regarding which LR is used, optimizer uses learning rate from inside the group : https://pytorch.org/docs/stable/_modules/torch/optim/adam.html
